lib/chrome.js

```diff
@@ -220,7 +220,11 @@ class Chrome extends EventEmitter {
                 if (this.secure) {
                     this.webSocketUrl = this.webSocketUrl.replace(/^ws:/i, 'wss:');
                 }
-                this._ws = new WebSocket(this.webSocketUrl);
+                this._ws = new WebSocket(this.webSocketUrl, [], {
+                    maxPayload: 256 * 1024 * 1024,
+                    perMessageDeflate: false,
+                    followRedirects: true,
```

Comment:
Why has `followRedirects` been set to `true`?

Reply:
Just in case someone is using a reverse proxy or some other intermediary in front of Chrome.